codec-http2/src/main/java/io/netty/handler/codec/http2/HpackHuffmanDecoder.java

```diff
@@ -64,183 +53,4656 @@
      * @throws Http2Exception EOS Decoded
      */
     public AsciiString decode(ByteBuf buf, int length) throws Http2Exception {
```

Comment:
In the current state of the code this method could be static

Reply:
It would be out of place, until the ctor is removed too.

Reply:
See https://github.com/netty/netty/pull/9307#discussion_r299329934

Reply:
Done.